### PR TITLE
Extract visual loading state setters into common methods

### LINKED ISSUE #133: Create function for setting app loading state
Move `document.body.classList.remove('loading');` and `document.body.classList.add('loading');` into (static) functions on the TopLevelApi.

### PR DESCRIPTION
**Pull request by sisou**
Resolves #133.

### CODE REVIEW

File: src/request/TopLevelApi.js [hunk 1]
```diff
@@ -118,9 +118,18 @@ class TopLevelApi { // eslint-disable-line no-unused-vars
...
     }
 
+    static setLoading() {
+        document.body.classList.add('loading');
+    }
+
+    static removeLoading() {
+        document.body.classList.remove('loading');
+    }
+
```
Comment (nibhar) [inline src/request/TopLevelApi.js]: As a suggestion for both in one function.
```javascript
static loading(loading=true) {
  document.body.classList.toggle('loading',loading);
}
```

Comment (Bettelstab) [reply src/request/TopLevelApi.js]: +1

Comment (Bettelstab) [reply src/request/TopLevelApi.js]: I would keep setLoading as name, though.

Comment (sisou) [reply src/request/TopLevelApi.js]: I disagree and actually made it two functions on purpose, to make it more explicit and not have to pass a boolean for removing it.

Comment (nibhar) [reply src/request/TopLevelApi.js]: I personally am in favour of one function for that kind of functionality. More so as `classList.toggle` exists.

**Pull request merged**